tests/integration/dhcp_test.py

```diff
@@ -540,7 +540,19 @@ def test_ipv6_dhcp_switch_on_to_off(dhcpcli_up):
     assert not _has_ipv6_auto_nameserver()
 
 
-def test_dhcp_on_bridge0(dhcpcli_up_with_delay_status):
+@pytest.fixture(scope='function')
+def disable_dhcpv6_server(dhcp_env):
```

Comment:
I guess this is explained by: `As the DHCPv6 using the IAID(different between interfaces) and DUID as
the client identifier, disabled the DHCPv6 before the 'test_dhcp_on_bridge0'
test.`
Right?

Reply:
Yes.